src/components/Person/Person.jsx

```diff
@@ -1 +1,12 @@
-// export const Person = ({ person }) => ();
+import '../../App.scss';
+
+export const Person = ({ person }) => (
```

Comment:
Try to use a destruction for `person`

Reply:
for some reason test wont see it even though its printesd correctly on the page. Tests showing that nothings is printed but it is so that is why i didnt use destruction